source/includes/mongodb-compatibility-table-java.rst

```diff
@@ -208,7 +208,9 @@
      -
      - |checkmark|
 
-.. [#driver-support] The 2.14 driver does not support all MongoDB 3.2 features (e.g.,
+.. [#3.12-driver-support]  The 3.12 driver does not support all MongoDB 4.4 features
```

Comment:
Nitpick: there's an extra space before "The". This doesn't cause errors or change the output, but just want to keep it in the expected format case something changes in the future.